### ISSUE TITLE
Model-anaylzer cannot load onnx model

### ISSUE THREAD

**Issue by sungyeon-neubla** [triton-inference-server/model_analyzer#606]
Our triton model serves onnx models and we would like to run model analyzer for them.
They are located in $(pwd)/triton-models/models and each model has one version each.

I ran the following docker command to launch the triton-server-sdk docker image:

`docker run -it --gpus all -v /var/run/docker.sock:/var/run/docker.sock -v $(pwd):/workspace/triton-models --net=host nvcr.io/nvidia/tritonserver:23.01-py3-sdk`

I ran the following command within the triton-server-sdk docker image:

`model-analyzer -v profile -f triton-models/model_analyzer_config.yaml --triton-launch-mode=docker --output-model-repository-path triton-models/output-model-repository/test-models --export-path triton-models/profile_results --override-output-model-repository`

With verbose mode on, I get the following output from the model-analyzer:

`09:35:06 [Model Analyzer] DEBUG: 
{'batch_sizes': [1],
 'checkpoint_directory': '/workspace/checkpoints',
 'client_max_retries': 50,
 'client_protocol': 'grpc',
 'collect_cpu_metrics': False,
 'concurrency': [],
 'config_file': 'triton-models/model_analyzer_config.yaml',
 'constraints': {},
 'duration_seconds': 3,
 'early_exit_enable': False,
 'export_path': 'triton-models/profile_results',
 'filename_model_gpu': 'metrics-model-gpu.csv',
 'filename_model_inference': 'metrics-model-inference.csv',
 'filename_server_only': 'metrics-server-only.csv',
 'gpu_output_fields': ['model_name',
                       'gpu_uuid',
                       'batch_size',
                       'concurrency',
                       'model_config_path',
                       'instance_group',
                       'satisfies_constraints',
                       'gpu_used_memory',
                       'gpu_utilization',
                       'gpu_power_usage'],
 'gpus': ['all'],
 'inference_output_fields': ['model_name',
                             'batch_size',
                             'concurrency',
                             'model_config_path',
                             'instance_group',
                             'max_batch_size',
                             'satisfies_constraints',
                             'perf_throughput',
                             'perf_latency_p99'],
 'latency_budget': None,
 'min_throughput': None,
 'model_repository': '/workspace/triton-models/models',
 'monitoring_interval': 1.0,
 'num_configs_per_model': 3,
 'num_top_model_configs': 0,
 'objectives': {'perf_throughput': 10},
 'output_model_repository_path': 'triton-models/output-model-repository/test-models',
 'override_output_model_repository': True,
 'perf_analyzer_cpu_util': 10240.0,
 'perf_analyzer_flags': {'input-data': 'zero'},
 'perf_analyzer_max_auto_adjusts': 10,
 'perf_analyzer_path': 'perf_analyzer',
 'perf_analyzer_timeout': 600,
 'perf_output': False,
 'perf_output_path': None,
 'plots': [{'name': 'throughput_v_latency', 'title': 'Throughput vs. Latency', 'x_axis': 'perf_latency_p99', 'y_axis': 'perf_throughput', 'monotonic': True},
           {'name': 'gpu_mem_v_latency', 'title': 'GPU Memory vs. Latency', 'x_axis': 'perf_latency_p99', 'y_axis': 'gpu_used_memory', 'monotonic': False}],
 'profile_models': [{'model_name': 'mhformer_onnx', 'cpu_only': False, 'objectives': {'perf_throughput': 10}, 'parameters': {'batch_sizes': [1], 'concurrency': []}, 'weighting': 1, 'perf_analyzer_flags': {'input-data': 'zero'}},
                    {'model_name': 'resnetv1d50_onnx', 'cpu_only': False, 'objectives': {'perf_throughput': 10}, 'parameters': {'batch_sizes': [1], 'concurrency': []}, 'weighting': 1, 'perf_analyzer_flags': {'input-data': 'zero'}},
                    {'model_name': 'yolov3_onnx', 'cpu_only': False, 'objectives': {'perf_throughput': 10}, 'parameters': {'batch_sizes': [1], 'concurrency': []}, 'weighting': 1, 'perf_analyzer_flags': {'input-data': 'zero'}}],
 'reload_model_disable': False,
 'run_config_profile_models_concurrently_enable': False,
 'run_config_search_disable': False,
 'run_config_search_max_concurrency': 1024,
 'run_config_search_max_instance_count': 5,
 'run_config_search_max_model_batch_size': 128,
 'run_config_search_min_concurrency': 1,
 'run_config_search_min_instance_count': 1,
 'run_config_search_min_model_batch_size': 1,
 'run_config_search_mode': 'brute',
 'server_output_fields': ['model_name',
                          'gpu_uuid',
                          'gpu_used_memory',
                          'gpu_utilization',
                          'gpu_power_usage'],
 'skip_summary_reports': False,
 'triton_docker_image': 'nvcr.io/nvidia/tritonserver:23.01-py3',
 'triton_docker_labels': {},
 'triton_docker_mounts': [],
 'triton_docker_shm_size': None,
 'triton_grpc_endpoint': 'localhost:8001',
 'triton_http_endpoint': 'localhost:8000',
 'triton_install_path': '/opt/tritonserver',
 'triton_launch_mode': 'docker',
 'triton_metrics_url': 'http://localhost:8002/metrics',
 'triton_output_path': None,
 'triton_server_environment': {},
 'triton_server_flags': {},
 'triton_server_path': 'tritonserver',
 'weighting': None}
09:35:06 [Model Analyzer] Initializing GPUDevice handles
09:35:07 [Model Analyzer] Using GPU 0 NVIDIA RTX A6000 with UUID GPU-925d4ceb-5adf-c3e5-a1f3-91bfb6b59783
09:35:07 [Model Analyzer] Using GPU 1 NVIDIA RTX A6000 with UUID GPU-bd65df87-5eb1-3a6b-7a7c-7df1fb64801e
09:35:07 [Model Analyzer] Using GPU 2 NVIDIA RTX A6000 with UUID GPU-7a42f7cd-5430-a4b0-f56c-50002487549a
09:35:07 [Model Analyzer] Using GPU 3 NVIDIA RTX A6000 with UUID GPU-59ce768c-b6a2-2bd1-2ebe-d3235d5332fa
09:35:07 [Model Analyzer] Using GPU 4 NVIDIA RTX A6000 with UUID GPU-c8a559f9-8ba7-8925-2f20-68b58640d8a7
09:35:07 [Model Analyzer] Using GPU 5 NVIDIA RTX A6000 with UUID GPU-f4a90d01-99f5-d0cf-fd31-9d40fc4348fe
09:35:07 [Model Analyzer] Using GPU 6 NVIDIA RTX A6000 with UUID GPU-51524c3a-8bd6-a018-5bc0-a6a4a49ac1bb
09:35:07 [Model Analyzer] Using GPU 7 NVIDIA RTX A6000 with UUID GPU-7984cc9e-496d-0554-db72-9cb76cd0dafc
09:35:07 [Model Analyzer] WARNING: Overriding the output model repo path "triton-models/output-model-repository/test-models"
09:35:07 [Model Analyzer] Starting a Triton Server using docker
09:35:07 [Model Analyzer] Pulling docker image nvcr.io/nvidia/tritonserver:23.01-py3
09:35:09 [Model Analyzer] Loaded checkpoint from file /workspace/checkpoints/1.ckpt
09:35:09 [Model Analyzer] GPU devices match checkpoint - skipping server metric acquisition
09:35:09 [Model Analyzer] 
09:35:09 [Model Analyzer] Creating model config: mhformer_onnx_config_default
09:35:09 [Model Analyzer] 
09:35:10 [Model Analyzer] DEBUG: Triton Server started.
09:35:18 [Model Analyzer] Model mhformer_onnx_config_default load failed: [StatusCode.INTERNAL] failed to load 'mhformer_onnx_config_default', failed to poll from model repository
09:35:23 [Model Analyzer] DEBUG: Stopped Triton Server.
09:35:23 [Model Analyzer] 
09:35:23 [Model Analyzer] Found existing model config: mhformer_onnx_config_0
09:35:23 [Model Analyzer]   Setting instance_group to [{'count': 1, 'kind': 'KIND_GPU'}]
09:35:23 [Model Analyzer]   Setting max_batch_size to 1
09:35:23 [Model Analyzer]   Enabling dynamic_batching
09:35:23 [Model Analyzer] 
09:35:24 [Model Analyzer] DEBUG: Triton Server started.
09:35:34 [Model Analyzer] Model mhformer_onnx_config_0 load failed: [StatusCode.INTERNAL] failed to load 'mhformer_onnx_config_0', failed to poll from model repository
09:35:40 [Model Analyzer] DEBUG: Stopped Triton Server.
09:35:40 [Model Analyzer] 
09:35:40 [Model Analyzer] Found existing model config: mhformer_onnx_config_1
09:35:40 [Model Analyzer]   Setting instance_group to [{'count': 2, 'kind': 'KIND_GPU'}]
09:35:40 [Model Analyzer]   Setting max_batch_size to 1
09:35:40 [Model Analyzer]   Enabling dynamic_batching
09:35:40 [Model Analyzer] 
09:35:40 [Model Analyzer] DEBUG: Triton Server started.
09:35:48 [Model Analyzer] Model mhformer_onnx_config_1 load failed: [StatusCode.INTERNAL] failed to load 'mhformer_onnx_config_1', failed to poll from model repository
09:36:09 [Model Analyzer] DEBUG: Stopped Triton Server.
09:36:09 [Model Analyzer] 
09:36:09 [Model Analyzer] Found existing model config: mhformer_onnx_config_2
09:36:09 [Model Analyzer]   Setting instance_group to [{'count': 3, 'kind': 'KIND_GPU'}]
09:36:09 [Model Analyzer]   Setting max_batch_size to 1
09:36:09 [Model Analyzer]   Enabling dynamic_batching
09:36:09 [Model Analyzer] 
09:36:14 [Model Analyzer] DEBUG: Triton Server started.
09:36:23 [Model Analyzer] Model mhformer_onnx_config_2 load failed: [StatusCode.INTERNAL] failed to load 'mhformer_onnx_config_2', failed to poll from model repository
09:36:28 [Model Analyzer] DEBUG: Stopped Triton Server.
09:36:28 [Model Analyzer] 
09:36:28 [Model Analyzer] Found existing model config: mhformer_onnx_config_3
09:36:28 [Model Analyzer]   Setting instance_group to [{'count': 4, 'kind': 'KIND_GPU'}]
09:36:28 [Model Analyzer]   Setting max_batch_size to 1
09:36:28 [Model Analyzer]   Enabling dynamic_batching
09:36:28 [Model Analyzer] 
09:36:29 [Model Analyzer] DEBUG: Triton Server started.
09:36:39 [Model Analyzer] Model mhformer_onnx_config_3 load failed: [StatusCode.INTERNAL] failed to load 'mhformer_onnx_config_3', failed to poll from model repository
09:36:44 [Model Analyzer] DEBUG: Stopped Triton Server.
09:36:44 [Model Analyzer] 
09:36:44 [Model Analyzer] Found existing model config: mhformer_onnx_config_4
09:36:44 [Model Analyzer]   Setting instance_group to [{'count': 5, 'kind': 'KIND_GPU'}]
09:36:44 [Model Analyzer]   Setting max_batch_size to 1
09:36:44 [Model Analyzer]   Enabling dynamic_batching
09:36:44 [Model Analyzer] 
09:36:44 [Model Analyzer] DEBUG: Triton Server started.
09:36:53 [Model Analyzer] Model mhformer_onnx_config_4 load failed: [StatusCode.INTERNAL] failed to load 'mhformer_onnx_config_4', failed to poll from model repository
09:36:58 [Model Analyzer] DEBUG: Stopped Triton Server.
09:36:58 [Model Analyzer] Saved checkpoint to /workspace/checkpoints/2.ckpt
09:36:58 [Model Analyzer] 
09:36:58 [Model Analyzer] Creating model config: resnetv1d50_onnx_config_default
09:36:58 [Model Analyzer] 
09:36:59 [Model Analyzer] DEBUG: Triton Server started.
09:37:08 [Model Analyzer] Model resnetv1d50_onnx_config_default load failed: [StatusCode.INTERNAL] failed to load 'resnetv1d50_onnx_config_default', failed to poll from model repository
09:37:13 [Model Analyzer] DEBUG: Stopped Triton Server.
09:37:13 [Model Analyzer] 
09:37:13 [Model Analyzer] Found existing model config: resnetv1d50_onnx_config_0
09:37:13 [Model Analyzer]   Setting instance_group to [{'count': 1, 'kind': 'KIND_GPU'}]
09:37:13 [Model Analyzer]   Setting max_batch_size to 1
09:37:13 [Model Analyzer]   Enabling dynamic_batching
09:37:13 [Model Analyzer] 
09:37:13 [Model Analyzer] DEBUG: Triton Server started.
09:37:23 [Model Analyzer] Model resnetv1d50_onnx_config_0 load failed: [StatusCode.INTERNAL] failed to load 'resnetv1d50_onnx_config_0', failed to poll from model repository
09:37:28 [Model Analyzer] DEBUG: Stopped Triton Server.
09:37:28 [Model Analyzer] 
09:37:28 [Model Analyzer] Found existing model config: resnetv1d50_onnx_config_1
09:37:28 [Model Analyzer]   Setting instance_group to [{'count': 2, 'kind': 'KIND_GPU'}]
09:37:28 [Model Analyzer]   Setting max_batch_size to 1
09:37:28 [Model Analyzer]   Enabling dynamic_batching
09:37:28 [Model Analyzer] 
09:37:29 [Model Analyzer] DEBUG: Triton Server started.
09:37:39 [Model Analyzer] Model resnetv1d50_onnx_config_1 load failed: [StatusCode.INTERNAL] failed to load 'resnetv1d50_onnx_config_1', failed to poll from model repository
09:37:44 [Model Analyzer] DEBUG: Stopped Triton Server.
09:37:44 [Model Analyzer] 
09:37:44 [Model Analyzer] Found existing model config: resnetv1d50_onnx_config_2
09:37:44 [Model Analyzer]   Setting instance_group to [{'count': 3, 'kind': 'KIND_GPU'}]
09:37:44 [Model Analyzer]   Setting max_batch_size to 1
09:37:44 [Model Analyzer]   Enabling dynamic_batching
09:37:44 [Model Analyzer] 
09:37:44 [Model Analyzer] DEBUG: Triton Server started.
09:37:53 [Model Analyzer] Model resnetv1d50_onnx_config_2 load failed: [StatusCode.INTERNAL] failed to load 'resnetv1d50_onnx_config_2', failed to poll from model repository
09:37:58 [Model Analyzer] DEBUG: Stopped Triton Server.
09:37:58 [Model Analyzer] 
09:37:58 [Model Analyzer] Found existing model config: resnetv1d50_onnx_config_3
09:37:58 [Model Analyzer]   Setting instance_group to [{'count': 4, 'kind': 'KIND_GPU'}]
09:37:58 [Model Analyzer]   Setting max_batch_size to 1
09:37:58 [Model Analyzer]   Enabling dynamic_batching
09:37:58 [Model Analyzer] 
09:37:58 [Model Analyzer] DEBUG: Triton Server started.
09:38:08 [Model Analyzer] Model resnetv1d50_onnx_config_3 load failed: [StatusCode.INTERNAL] failed to load 'resnetv1d50_onnx_config_3', failed to poll from model repository
09:38:13 [Model Analyzer] DEBUG: Stopped Triton Server.
09:38:13 [Model Analyzer] 
09:38:13 [Model Analyzer] Found existing model config: resnetv1d50_onnx_config_4
09:38:13 [Model Analyzer]   Setting instance_group to [{'count': 5, 'kind': 'KIND_GPU'}]
09:38:13 [Model Analyzer]   Setting max_batch_size to 1
09:38:13 [Model Analyzer]   Enabling dynamic_batching
09:38:13 [Model Analyzer] 
09:38:14 [Model Analyzer] DEBUG: Triton Server started.
09:38:24 [Model Analyzer] Model resnetv1d50_onnx_config_4 load failed: [StatusCode.INTERNAL] failed to load 'resnetv1d50_onnx_config_4', failed to poll from model repository
09:38:29 [Model Analyzer] DEBUG: Stopped Triton Server.
09:38:29 [Model Analyzer] Saved checkpoint to /workspace/checkpoints/3.ckpt
09:38:29 [Model Analyzer] 
09:38:29 [Model Analyzer] Creating model config: yolov3_onnx_config_default
09:38:29 [Model Analyzer] 
09:38:30 [Model Analyzer] DEBUG: Triton Server started.
09:38:39 [Model Analyzer] Model yolov3_onnx_config_default load failed: [StatusCode.INTERNAL] failed to load 'yolov3_onnx_config_default', failed to poll from model repository
09:38:43 [Model Analyzer] DEBUG: Stopped Triton Server.
09:38:43 [Model Analyzer] 
09:38:43 [Model Analyzer] Creating model config: yolov3_onnx_config_0
09:38:43 [Model Analyzer]   Setting instance_group to [{'count': 1, 'kind': 'KIND_GPU'}]
09:38:43 [Model Analyzer]   Setting max_batch_size to 1
09:38:43 [Model Analyzer]   Enabling dynamic_batching
09:38:43 [Model Analyzer] 
09:38:43 [Model Analyzer] DEBUG: Triton Server started.
09:38:53 [Model Analyzer] Model yolov3_onnx_config_0 load failed: [StatusCode.INTERNAL] failed to load 'yolov3_onnx_config_0', failed to poll from model repository
09:38:58 [Model Analyzer] DEBUG: Stopped Triton Server.
09:38:58 [Model Analyzer] 
09:38:58 [Model Analyzer] Creating model config: yolov3_onnx_config_1
09:38:58 [Model Analyzer]   Setting instance_group to [{'count': 2, 'kind': 'KIND_GPU'}]
09:38:58 [Model Analyzer]   Setting max_batch_size to 1
09:38:58 [Model Analyzer]   Enabling dynamic_batching
09:38:58 [Model Analyzer] 
09:38:59 [Model Analyzer] DEBUG: Triton Server started.
09:39:09 [Model Analyzer] Model yolov3_onnx_config_1 load failed: [StatusCode.INTERNAL] failed to load 'yolov3_onnx_config_1', failed to poll from model repository
09:39:14 [Model Analyzer] DEBUG: Stopped Triton Server.
09:39:14 [Model Analyzer] 
09:39:14 [Model Analyzer] Creating model config: yolov3_onnx_config_2
09:39:14 [Model Analyzer]   Setting instance_group to [{'count': 3, 'kind': 'KIND_GPU'}]
09:39:14 [Model Analyzer]   Setting max_batch_size to 1
09:39:14 [Model Analyzer]   Enabling dynamic_batching
09:39:14 [Model Analyzer] 
09:39:14 [Model Analyzer] DEBUG: Triton Server started.
09:39:23 [Model Analyzer] Model yolov3_onnx_config_2 load failed: [StatusCode.INTERNAL] failed to load 'yolov3_onnx_config_2', failed to poll from model repository
09:39:28 [Model Analyzer] DEBUG: Stopped Triton Server.
09:39:28 [Model Analyzer] 
09:39:28 [Model Analyzer] Creating model config: yolov3_onnx_config_3
09:39:28 [Model Analyzer]   Setting instance_group to [{'count': 4, 'kind': 'KIND_GPU'}]
09:39:28 [Model Analyzer]   Setting max_batch_size to 1
09:39:28 [Model Analyzer]   Enabling dynamic_batching
09:39:28 [Model Analyzer] 
09:39:28 [Model Analyzer] DEBUG: Triton Server started.
09:39:38 [Model Analyzer] Model yolov3_onnx_config_3 load failed: [StatusCode.INTERNAL] failed to load 'yolov3_onnx_config_3', failed to poll from model repository
09:39:43 [Model Analyzer] DEBUG: Stopped Triton Server.
09:39:43 [Model Analyzer] 
09:39:43 [Model Analyzer] Creating model config: yolov3_onnx_config_4
09:39:43 [Model Analyzer]   Setting instance_group to [{'count': 5, 'kind': 'KIND_GPU'}]
09:39:43 [Model Analyzer]   Setting max_batch_size to 1
09:39:43 [Model Analyzer]   Enabling dynamic_batching
09:39:43 [Model Analyzer] 
09:39:44 [Model Analyzer] DEBUG: Triton Server started.
09:39:53 [Model Analyzer] Model yolov3_onnx_config_4 load failed: [StatusCode.INTERNAL] failed to load 'yolov3_onnx_config_4', failed to poll from model repository
09:39:58 [Model Analyzer] DEBUG: Stopped Triton Server.
09:39:58 [Model Analyzer] Saved checkpoint to /workspace/checkpoints/4.ckpt
09:39:58 [Model Analyzer] Profile complete. Profiled 0 configurations for models: []
09:39:58 [Model Analyzer] 
09:39:58 [Model Analyzer] WARNING: Inference output field "perf_throughput", has no data
09:39:58 [Model Analyzer] WARNING: Inference output field "perf_latency_p99", has no data
09:39:58 [Model Analyzer] Exporting server only metrics to triton-models/profile_results/results/metrics-server-only.csv
Models (Inference):
Model   Batch   Concurrency   Model Config Path   Instance Group   Max Batch Size   Satisfies Constraints  

Models (GPU Metrics):
Model   GPU UUID   Batch   Concurrency   Model Config Path   Instance Group   Satisfies Constraints   GPU Memory Usage (MB)   GPU Utilization (%)   GPU Power Usage (W)  

Server Only:
Model           GPU UUID                                   GPU Memory Usage (MB)   GPU Utilization (%)   GPU Power Usage (W)  
triton-server   GPU-7984cc9e-496d-0554-db72-9cb76cd0dafc   1092.0                  0.0                   63.8                 
triton-server   GPU-59ce768c-b6a2-2bd1-2ebe-d3235d5332fa   1092.0                  0.0                   71.5                 
triton-server   GPU-c8a559f9-8ba7-8925-2f20-68b58640d8a7   1092.0                  0.0                   70.9                 
triton-server   GPU-bd65df87-5eb1-3a6b-7a7c-7df1fb64801e   1092.0                  0.0                   76.0                 
triton-server   GPU-7a42f7cd-5430-a4b0-f56c-50002487549a   1092.0                  0.0                   72.6                 
triton-server   GPU-925d4ceb-5adf-c3e5-a1f3-91bfb6b59783   1092.0                  0.0                   66.8                 
triton-server   GPU-f4a90d01-99f5-d0cf-fd31-9d40fc4348fe   1092.0                  0.0                   65.6                 
triton-server   GPU-51524c3a-8bd6-a018-5bc0-a6a4a49ac1bb   1092.0                  0.0                   73.1                 

09:39:58 [Model Analyzer] WARNING: Requested top 3 configs, but none satisfied constraints. Showing available constraint failing configs for this model.
09:39:58 [Model Analyzer] WARNING: Requested top 3 configs, but found only 0. Showing all available configs for this model.
09:39:58 [Model Analyzer] WARNING: Requested top 3 configs, but none satisfied constraints. Showing available constraint failing configs for this model.
09:39:58 [Model Analyzer] WARNING: Requested top 3 configs, but found only 0. Showing all available configs for this model.
09:39:58 [Model Analyzer] WARNING: Requested top 3 configs, but none satisfied constraints. Showing available constraint failing configs for this model.
09:39:58 [Model Analyzer] WARNING: Requested top 3 configs, but found only 0. Showing all available configs for this model.
09:39:58 [Model Analyzer] WARNING: Requested top 3 configs, but none satisfied constraints. Showing available constraint failing configs for this model.
09:39:58 [Model Analyzer] WARNING: Requested top 3 configs, but found only 0. Showing all available configs for this model.
09:39:58 [Model Analyzer] WARNING: Requested top 3 configs, but none satisfied constraints. Showing available constraint failing configs for this model.
09:39:58 [Model Analyzer] WARNING: Requested top 3 configs, but found only 0. Showing all available configs for this model.
09:39:58 [Model Analyzer] WARNING: Requested top 3 configs, but none satisfied constraints. Showing available constraint failing configs for this model.
09:39:58 [Model Analyzer] WARNING: Requested top 3 configs, but found only 0. Showing all available configs for this model.
09:39:58 [Model Analyzer] WARNING: Requested top 3 configs, but none satisfied constraints. Showing available constraint failing configs for this model.
09:39:58 [Model Analyzer] WARNING: Requested top 3 configs, but found only 0. Showing all available configs for this model.
09:39:58 [Model Analyzer] WARNING: Requested top 3 configs, but none satisfied constraints. Showing available constraint failing configs for this model.
09:39:58 [Model Analyzer] WARNING: Requested top 3 configs, but found only 0. Showing all available configs for this model.
09:39:58 [Model Analyzer] WARNING: Requested top 3 configs, but none satisfied constraints. Showing available constraint failing configs for this model.
09:39:58 [Model Analyzer] WARNING: Requested top 3 configs, but found only 0. Showing all available configs for this model.
09:39:58 [Model Analyzer] WARNING: No data found for model mhformer_onnx, skipping export summary.
09:39:58 [Model Analyzer] WARNING: No data found for model resnetv1d50_onnx, skipping export summary.
09:39:58 [Model Analyzer] WARNING: No data found for model yolov3_onnx, skipping export summary.
09:39:58 [Model Analyzer] WARNING: Requested top 3 configs, but none satisfied constraints. Showing available constraint failing configs for this model.
09:39:58 [Model Analyzer] WARNING: Requested top 3 configs, but found only 0. Showing all available configs for this model.
09:39:58 [Model Analyzer] WARNING: Requested top 3 configs, but none satisfied constraints. Showing available constraint failing configs for this model.
09:39:58 [Model Analyzer] WARNING: Requested top 3 configs, but found only 0. Showing all available configs for this model.
09:39:58 [Model Analyzer] To generate detailed reports for the 0 best configurations, run `model-analyzer report --report-model-configs  --export-path triton-models/profile_results --config-file triton-models/model_analyzer_config.yaml``

From what I can see from the log, it complains that it cannot load the model with error code StatusCode.INTERNAL which does not quite help for debugging the issue.

I also do have some BLS python backend models that I would like to run Model-analyzer for but I saw some past issue stating that it is not supported yet. Is it still the case? Is it possible to share the expected timelines?

Thank you in advance!

**Comment by tgerdesnv** [comment]
>From what I can see from the log, it complains that it cannot load the model with error code StatusCode.INTERNAL which does not quite help for debugging the issue.

Agreed. We are working on better parsing of the triton logs to provide a better error message. For now, your best bet is to run with the following model analyzer option and look through the triton log to see why it wasn't able to load:

```
  --triton-output-path TRITON_OUTPUT_PATH
                        The full path to the file to which Triton server instance will append their log output. If not
                        specified, they are not written.
```

> I also do have some BLS python backend models that I would like to run Model-analyzer for but I saw some past issue stating that it is not supported yet. Is it still the case? Is it possible to share the expected timelines?

Ensemble support was just added and will be available in 23.02. We are starting work on BLS support in Model Analyzer and hope to have it ready in the next couple of releases.

@nv-braf

**Comment by nv-braf** [comment]
BLS is now available in the 23.04 release.

**Comment by dyastremsky** [comment]
This feature is now available. Closing this issue.
